### PR TITLE
add ability to customize license prompt text via --licensePrompt option + README.md example

### PR DESCRIPTION
**Pull request by ek9**
add ability to customize license prompt text via --licensePrompt parameter or option

Update README.md example

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -44,6 +44,13 @@ module.exports = class GeneratorLicense extends Generator {
       defaults: (new Date()).getFullYear()
     });
 
+    this.option('licensePrompt', {
```
Comment (SBoudrias) [inline app/index.js]: Let's set a `default` value here instead of doing a if check later on.

**Comment by SBoudrias** [comment]
What is the use case for this feature?

**Comment by ek9** [comment]
Updated according to request.

Use case: I want to use this as a sub-generator. I use it together with other license generator, so this results in 2 prompts being asked to pick 2 different licenses. It becomes rather confusing as I cannot control when the prompts fire, so tailoring a license prompt text message is the only way I can make it useful.

File: app/index.js (hunk 1 above)
Comment (SBoudrias) [inline app/index.js]: Let's declare as `hide: true` to not bloat the end user help prompt. Documenting this option like you did in the readme should be enough.

**Comment by ek9** [comment]
Done. I've also changed the `required:` to `false`.

**Comment by SBoudrias** [comment]
Thanks!

**Comment by ek9** [comment]
Thanks for merging!

**Pull request merged**